source/simplex/quadrature_lib.cc

```diff
@@ -82,6 +82,44 @@ namespace Simplex
             this->weights.emplace_back(q12 * 0.132394152789);
             this->weights.emplace_back(q12 * 0.132394152789);
           }
+        else if (n_points_1D == 4)
+          {
+            const double q12 = 0.5;
+
+            // clang-format off
+            this->quadrature_points.emplace_back(0.06493051315916486, 0.06493051315916486);
```

Comment:
On my TODO list here is to deconstruct https://github.com/nschloe/quadpy/blob/main/quadpy/t3/_witherden_vincent/witherden_vincent_07.json so that we don't have to copy and paste points.